client/test/e2e/debtor_groups/debtor_groups.spec.js

```diff
@@ -0,0 +1,126 @@
+/* jshint expr: true */
+/* element, by, inject, browser */
+
+var chai    = require('chai');
+var expect  = chai.expect;
+var helpers = require('../shared/helpers');
+
+helpers.configure(chai);
+
+var FormUtils = require('../shared/FormUtils');
+var components = require('../shared/components');
+
+describe('Debtor Groups tests suit :: ', function () {
+
+  var PATH = '#/debtor_groups';
+
+  var debtorGroup = {
+    name : '(E2E) Debtor Group',
+    account_id : 3638,
+    location_id : '03a329b2-03fe-4f73-b40f-56a2870cc7e6',
+    phone : '0811838662',
+    email : 'debtorgroup@info.com',
+    note : '(E2E) Nouveau debtor group de test',
+    max_credit : 2500
+  };
+
+  var update = {
+    name : '(E2E) Debtor Group (updated)',
+    account_id : 3638,
+    location_id : '03a329b2-03fe-4f73-b40f-56a2870cc7e6',
+    phone : '0818061031',
+    email : 'debtorgroup@info.com (updated)',
+    note : '(E2E) Nouveau debtor group de test (updated)',
+    max_credit : 5500
+  };
+
+  /** navigate to the debtor group module */
+  beforeEach(function () {
+    browser.get(PATH);
+  });
+
+  it('successfully creates a new Debtor Group', function () {
+    FormUtils.buttons.create();
+    /** debtor group info */
+    FormUtils.input('DebtorGroupCtrl.debtorGroup.name', debtorGroup.name);
+    FormUtils.input('DebtorGroupCtrl.debtorGroup.account_id', debtorGroup.account_id);
+    element.all(by.options('price.uuid as price.label for price in DebtorGroupCtrl.prices.data')).get(1).click();
```

Comment:
In the future, it might be helpful to include a comment describing what price list you are selecting.